### PR TITLE
graphite - plugin for querying Graphite metric store system

### PR DESCRIPTION
**Pull request by pawellesniewski**

### CODE REVIEW

**Comment by ch40s3k** [comment]
:+1:

File: alamo_worker/plugins/graphite.py [hunk 1]
```diff
@@ -11,6 +13,77 @@
...
+                    'format': 'json'
+                }
+            )
+            response.raise_for_status()
+            data = response.json()
+
+            result = self.process(check, data)
+
+        except (ValueError, RequestException) as e:
+            logger.error(e)
```
Comment (deejay1) [inline alamo_worker/plugins/graphite.py]: shouldn't we do an "unknown" result in that case?

Comment (pawellesniewski) [inline alamo_worker/plugins/graphite.py]: fixed

**Comment by deejay1** [comment]
:+1:

**Comment by deejay1** [comment]
Merge after one +1 ? Naughty :P :palm_tree:

**Pull request merged**